pkgs/applications/editors/emacs-25/default.nix

```diff
@@ -81,18 +76,15 @@ stdenv.mkDerivation rec {
     mv nextstep/Emacs.app $out/Applications
   '';
 
-  doCheck = !stdenv.isDarwin;
+  doCheck = false;
```

Comment:
Can you please add a link to https://github.com/NixOS/nixpkgs/issues/13573 here so that people know why the test suite is disabled?